### DISCUSSION TITLE
新版的内核文件是不是少了armv8.的？

### DISCUSSION THREAD

**Discussion by xiaiohuan** [vernesong/OpenClash#2827] [Q&A]
如题 新版少了相关文件下载现在内核都没法下载手动下载直接404 找到源码文件看 3个内核文件里面都没有armv8构架的文件下载

**Comment by hl8686** [comment]
我也觉得很奇怪，一直无法启动，r4是， armv8内核的

**Reply by xiaiohuan** [reply]
现在不止这个问题 现在这个文件问题知道原因了 是合并ram64通用了 现在新问题就是现在的这个插件新版本 在AX6 上面跑编译时直接编译这个最新版本 出来的固件会导致无法启动 一直闪灯 退回之前的版本编译 就能立马启动 不知道是咋回事

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/89022914/202976872-1dd2eac5-68fa-4fd6-a7d5-d7f885bcb7b8.png)

**Reply by xiaiohuan** [reply]
合并通用用吗

**Reply by ghost** [reply]
改名而已，现在和clash官方命名一致

**Reply by xiaiohuan** [reply]
嗯好的知道已经手动下载上传上去了 可以用 谢谢

**Reply by ghost** [reply]
OKK，甭客气
